rpc/openapi/openapi.yaml

```diff
@@ -262,10 +252,78 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
 
+  /events:
```

Comment:
Does events not have any parameters?

Reply:
Haha, whoops! Good catch, I missed a commit.